pycheribuild/projects/fvp_firmware.py

```diff
@@ -0,0 +1,264 @@
+#
```

Comment:
Should this not be a morello_firware.py given the FVP and Morello board are supposed to be very similar? (In theory I assume they're meant to be able to run basically the same firmware, but in practice I'm sure that's not actually the case...)

Reply:
Yeah they should be the same but until I know for sure I'll keep the name FVP. I believe some of the configurations are a bit different for FVP.